src/components/saveAccountButton/saveAccountButton.js

```diff
@@ -0,0 +1,33 @@
+import { MenuItem } from 'react-toolbox/lib/menu';
+import React from 'react';
+
+import { getSavedAccount, removeSavedAccount } from '../../utils/saveAccount';
+import SaveAccount from '../saveAccount';
+
+export default class SaveAccountButton extends React.Component {
+  removeSavedAccount() {
+    removeSavedAccount();
+    this.props.successToast({ label: 'Account forgotten locally.' });
+    this.forceUpdate();
+  }
+
+  render() {
+    return (getSavedAccount() ?
+      <MenuItem caption="Forget this account locally"
```

Comment:
Maybe _Forget this account_? since we don't use the words _remove_ or _delete_, adding the adverb _locally_ may create confusion. (This is just a suggestion)